charts/tileserver-api/templates/deployment.yaml

```
@@ -0,0 +1,85 @@
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "tileserver.name" . }}
+  labels:
+    {{- include "api.labels" . | nindent 4 }}
```

Стоит дать возможно выставить дополнительные label'ы.
```suggestion
  {{- with .Values.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
  labels:
    {{- include "api.labels" . | nindent 4 }}
  {{- with .Values.labels }}
    {{- toYaml . | nindent 4 }}
  {{- end }}
```